docker/puppetserver-standalone/spec/puppetserver_spec.rb

```diff
@@ -0,0 +1,56 @@
+#! /usr/bin/env ruby
+
+require 'rspec/core'
+
+describe 'puppetserver-standalone container' do
+  def puppetserver_health_check(container)
+    status = %x(docker inspect "#{container}" --format '{{.State.Health.Status}}').chomp
+    STDOUT.puts "queried health status of #{container}: #{status}"
+    return status
+  end
+
+  before(:all) do
+    %x(docker pull puppet/puppet-agent-alpine:latest)
+    @image = ENV['PUPPET_TEST_DOCKER_IMAGE']
```

Comment:
Where is this value being set from? Calculated inside `puppet_docker_tools` or something?

Reply:
yeah, that's calculated in `puppet_docker_tools` https://github.com/puppetlabs/puppet_docker_tools/search?q=PUPPET_TEST_DOCKER_IMAGE&unscoped_q=PUPPET_TEST_DOCKER_IMAGE

In moving to Makefile, it just gets set there before calling the tests.